templates/zerver/help/delete-a-message.md

```diff
@@ -0,0 +1,7 @@
+# Delete a message
+
+{!admin-only.md!}
+{!message-actions.md!}
+{!down-chevron.md!}
+
+2. Select the **Delete message** option from the dropdown to delete that message.
```

Comment:
I like your moving this into the existing article.  How about we just move this to be a subsection of "edit or delete a message"  at `###` depth rather than its own page?  I think that'd be better, since now if you come across this page, you don't get that briefing.